src/cytodataframe/volume.py

```
@@ -0,0 +1,374 @@
+        "document.head.appendChild(script);"
+        "}else{"
+        "const wait=setInterval(function(){"
+        "if(window.vtk){clearInterval(wait);initAll();}"
+        "},50);"
+        "setTimeout(function(){clearInterval(wait);},10000);"
+        "}"
+        "})();"
+    )
+
+
+def extract_volume_from_ome_arrow(  # noqa: C901, PLR0912
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `C901`, `PLR0912`)

Remove unused `noqa` directive

(RUF100)